spec/features/merchants/dashboard/items_index_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'rails_helper'
+
+RSpec.describe "Merchant Items Index", type: :feature do
+  context "as a merchant" do
+    before(:each) do
+      @merchant = create(:merchant)
+
+      @other_merchant_item = create(:item)
```

Comment:
Question: Does @other_merchant_item need @other_merchant object in order to be created and for that item to be associated with @other_merchant.

Reply:
@hsmitha26 I don't care what merchant @other_merchant_item is associated with. It will just be some default from FactoryBot (it won't be @merchant, and that's all I care about)! Does that make sense?